File: IAIso-v5.0/core/iaiso-java/iaiso-audit/src/main/java/io/iaiso/audit/StdoutSink.java
package io.iaiso.audit;

/** A sink that writes one JSON event per line to {@code System.out}. */
public final class StdoutSink implements Sink {
    public static final StdoutSink INSTANCE = new StdoutSink();
    @Override
    public void emit(Event event) {
        System.out.println(event.toJson());
    }
}

File: IAIso-v5.0/core/iaiso-php/src/Audit/StdoutSink.php
<?php

declare(strict_types=1);

namespace IAIso\Audit;

/** Writes one JSON object per event to STDOUT. */
final class StdoutSink implements Sink
{
    private static ?StdoutSink $instance = null;

    public static function instance(): self
    {
        return self::$instance ??= new self();
    }

    public function emit(Event $event): void
    {
        echo $event->toJson() . PHP_EOL;
    }
}
